components/poll.js

```diff
@@ -14,6 +14,13 @@ export default function Poll ({ item }) {
   const { me } = useMe()
   const pollVote = usePollVote({ query: POLL_VOTE, itemId: item.id })
   const toaster = useToast()
+  const shuffleOptions = (options) => {
+    return options
+      .map(value => ({ value, sort: Math.random() }))
+      .sort((a, b) => a.sort - b.sort)
+      .map(({ value }) => value)
+  }
```

Comment:
```suggestion
  const shuffleOptions = (options) => {
    // shuffle copy because item.poll.options is read-only
    return [...options].sort(() => Math.random() - 0.5)
  }
```

Reply:
ok, I'll give this a go

Reply:
I think I've addressed everything you've brought up here. @ekzyis